### DISCUSSION TITLE
JC LazyColumn increasing RAM usage

### DISCUSSION THREAD

**Discussion by Garneg** [coil-kt/coil#1776] [Q&A]
I use Compose LazyColumn. Each item has `AsyncImage`. The more i scroll, the more ram my application uses. I tried to set both `memoryCachePolicy` and `diskCachePolicy` to disabled, but it didn't do a thing, memory usage kept going up. Clearing cache with `memoryCache.clear()` and `diskCache.clear()` not working too.  (It's 100% problem on the Coil side. I already tried to remove AsyncImage from item, and ram was fine). May be i am missing something that solves this problem

**Comment by colinrtwhite** [comment]
This sounds like https://issuetracker.google.com/issues/240975572, which is fixed in `androidx.compose.ui:ui:1.5.0-alpha03`.

**Reply by Garneg** [reply]
Cool. I have solved this problem. Somehow GC is not cleaning memory after item with AsyncImage leaves the viewport if contentType of items() is set. Without this paremeter everything works fine. It's still a bug, but not that critical. Thank you for the answer!
